tests/e2e/utils/workspace/ITestWorkspaceUtil.ts

```diff
@@ -12,15 +12,68 @@ import { WorkspaceStatus } from './WorkspaceStatus';
 import { che } from '@eclipse-che/api';
 
 export interface ITestWorkspaceUtil {
+    waitWorkspaceStatus(namespace: string, workspaceName: string, expectedWorkspaceStatus: WorkspaceStatus) : void;
+    stopWorkspaceByName(workspaceName: string) : void;
+    /**
+     * Delete a worksapce without stopping phase (similar with force deleting)
+     */
+    deleteWorkspaceByName(workspaceName: string) : void;
+    /**
+     * Stop workspace before deleting with checking stopping phase
+     */
+    stopAndDeleteWorkspaceByName(workspaceName: string) : void;
+    /**
```

Comment:
It's hard to read, without empty lines between the method declarations.

Reply:
fixed

Reply:
thanks!